src/scss/components/_button.scss

```diff
@@ -222,4 +208,46 @@ $button-padding-h: $grid-margin-2;
       color: var(--of-button-danger-bg);
     }
   }
+
+  @include modifier('anchor') {
```

Comment:
I can't easily tell from the diff - but why was this broken out of the `button-theme` mixin? Is it possible to add a `@content` to that button and include additional styles here as part of the mixin call then?

e.g.

```scss
@include button-theme('anchor', ...) {
  additional-property: value;
}
```

Reply:
When we added the "image" button we said that we wanted to move out of the "button-theme". So, since I was modifying it, I thought I'd refactor it so that we have fewer `!important`

Reply:
hrm, I think it'd be nicer to have a single refactor PR to tackle _all_ of them, in this in-between state I think it increases confusion for Jiro for example who's working on accessibility things related to buttons, amongst other :thinking:

Reply:
Ok, I will make a PR with the refactor separately from this PR!

Reply:
https://github.com/open-formulieren/open-forms-sdk/issues/339